docs/csharp/language-reference/attributes/general.md

```diff
@@ -58,15 +65,19 @@ The string provided as the first argument to the attribute constructor is displa
 
 :::code language="csharp" source="snippets/ObsoleteExample.cs" id="Snippet2" :::
 
-## `Experimental` attribute
+The Windows Foundation Metadata libraries use the <xref:Windows.Foundation.Metadata.DeprecatedAttribute?displayProperty=nameWithType> instead of the `ObsoleteAttribute`.
+
+## `Experimental` attributes
 
 Beginning in C# 12, types, methods, and assemblies can be marked with the <xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute?displayProperty=nameWithType> to indicate an experimental feature. The compiler issues a warning if you access a method or type annotated with the <xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>. All types declared in an assembly or module marked with the `Experimental` attribute are experimental. The compiler issues a warning if you access any of them. You can disable these warnings to pilot an experimental feature.
 
 > [!WARNING]
-> Experimental features are subject to changes. The APIs may change, or they may be removed in future updates. Including experimental features is a way for library authors to get feedback on ideas and concepts for future development. Use extreme caution when using any feature marked as experimental.
+> Experimental features are subject to changes. The APIs can change, or they can be removed in future updates. Including experimental features is a way for library authors to get feedback on ideas and concepts for future development. Use extreme caution when using any feature marked as experimental.
```

Comment:
Once https://github.com/dotnet/docs/pull/44436 is merged, we should interlink between https://learn.microsoft.com/en-us/dotnet/fundamentals/runtime-libraries/preview-apis#experimentalattribute and here.

Reply:
I responded to all the other feedback. Should I merge first, or do you want to?

Reply:
I can't merge until I get an approving review.

Reply:
I reviewed and approved it.